src/BlobField/blobs.js

```diff
@@ -0,0 +1,210 @@
+		return this.balls[idx];
+	}
+
+	calcRadius(idx) {
+		const viewArea =
+			paper.view.size.width * paper.view.size.height * this.viewRatio;
+		let radius;
+		if (this.collapsed && idx != 0) {
+			let orentation = paper.view.size.width > paper.view.size.height;
+			let totalLength = orentation ? paper.view.size.height : paper.view.size.width;
+			radius = totalLength / (this.numBalls * 2);
+		} else {
+			radius = Math.sqrt(viewArea / this.numBalls / Math.PI);
+			radius += Math.random() * (this.squeezeFactor * radius);
+		}
+		return radius;
+	}
+
+	setup() {
+		const mouseBall = new Ball(
+			this.calcRadius(0),
+			new paper.Point(0, 0),
+			new paper.Point(0, 0)
+		);
+		mouseBall.radius = this.calcRadius(0) * this.mouseRadiusMultiplier;
+		mouseBall.path.opacity = 0;
+		mouseBall.path.isMouse = true;
+		mouseBall.isVertical = paper.view.size.width > paper.view.size.height;
+		mouseBall.setIdx(0);
+		this.balls.push(mouseBall);
+
+		for (let i = 0; i < this.numBalls; i++) {
+			const position = paper.Point.random().multiply(paper.view.size);
+			const force = new paper.Point({
+				angle: 1 * Math.random(),
+				length: Math.random() * 10,
+			});
+			const currBall = new Ball(this.calcRadius(i), position, force);
+			currBall.path.opacity = this.opacity;
+			currBall.shadowColor.alpha = this.opacity / 2;
+			currBall.path.artist = this.artists[i];
+			currBall.setIdx(this.balls.length);
+			currBall.isVertical = paper.view.size.width > paper.view.size.height;
+			currBall.path.onMouseEnter = this.pathOnMouseEnter.bind(this);
+			currBall.path.onMouseLeave = this.pathOnMouseLeave.bind(this);
+			currBall.path.onClick = this.pathOnClick.bind(this);
+			this.balls.push(currBall);
+		}
+	}
+
+	onFrame() {
+		for (let i = 0; i < this.balls.length - 1; i++)
+			for (let j = i + 1; j < this.balls.length; j++)
+				this.balls[i].react(this.balls[j]);
+
+		for (let i = 1; i < this.balls.length; i++) {
+			this.balls[i].iterate();
+			this.balls[i].updateColor();
+		}
+
+		this.balls[0].updateShape();
+		this.balls[0].updateColor();
+
+		// Mouse Easing
+		let easeFactor = this.defaultEasing;
+		if (this.isSlowSim) {
+			easeFactor = this.slowSimEasing;
+		}
+		const dx = this.mouseTargetX - this.mouseCurrX;
+		this.mouseCurrX += dx * easeFactor;
+		const dy = this.mouseTargetY - this.mouseCurrY;
+		this.mouseCurrY += dy * easeFactor;
+
+		// Move the hidden/mouse blob
+		if (this.collapsed) {
+			this.balls[0].point = new paper.Point(
+				-this.balls[0].radius,
+				-this.balls[0].radius
+			);
+		} else {
+			this.balls[0].point = new paper.Point(this.mouseCurrX, this.mouseCurrY);
+		}
+	}
+
+	onResize() {
+		for (let i = 0; i < this.balls.length; i++) {
+			this.balls[i].radius = this.calcRadius(i);
+			let tempIsVert = paper.view.size.width > paper.view.size.height;
+			this.balls[i].isVertical = tempIsVert;
+		}
+		this.balls[0].radius = this.calcRadius(0) * this.mouseRadiusMultiplier;
+	}
+
+	onMouseMove(x,y) {
+		this.mouseTargetX = x;
```

Comment:
@hsab is there a reason changing the signature of this fn would mess up cursor placement? I adjusted it because react's event objects look a bit different than paper's

Reply:
I think the issue right now is that event does not have a mouseX or mouseY property. 
Changed ln 36 of index.js to the following fixes the issue for me:
```
const handleMouseMove = (evt) => {
          blobs.onMouseMove(evt.clientX, evt.clientY);
};
```